components/eamxx/src/physics/rrtmgp/atmosphere_radiation.cpp

```diff
@@ -845,7 +846,12 @@ void RRTMGPRadiation::run_impl (const int dt) {
     const int kbot = nlay+1;
 
     // Compute diffuse flux as difference between total and direct; use YAKL parallel_for here because these are YAKL objects
-    parallel_for(Bounds<3>(nswbands,nlay+1,ncol), YAKL_LAMBDA(int ibnd, int ilev, int icol) {
```

Comment:
Since this is no longer a YAKL parallel_for, we should edit the comment that says “use YAKL parallel for”